consensus/common/src/test/java/org/hyperledger/besu/consensus/common/bft/FrequentMessageMulticasterTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright contributors to Besu.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.consensus.common.bft;
+
+import static org.mockito.Mockito.atLeast;
+import static org.mockito.Mockito.atMost;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import org.hyperledger.besu.consensus.common.bft.network.ValidatorMulticaster;
+import org.hyperledger.besu.ethereum.p2p.rlpx.wire.MessageData;
+
+import java.util.Collections;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FrequentMessageMulticasterTest {
+  private FrequentMessageMulticaster multicaster;
+  private ValidatorMulticaster mockValidatorMulticaster;
+
+  @BeforeEach
+  void setUp() {
+    mockValidatorMulticaster = mock(ValidatorMulticaster.class);
+    multicaster = new FrequentMessageMulticaster(mockValidatorMulticaster, 5000);
```

Comment:
Better to use a smaller interval for testing to decrease the test time taken. Maye 1000 ms instead?